app/models/product.rb

```diff
@@ -60,16 +60,16 @@ class Product < ActiveRecord::Base
   scope :mergeable_into_order, -> { not_archived.where(type: mergeable_types) }
 
   def self.types
-    @types ||= [Instrument, Item, Service, Bundle]
+    @types ||= [Instrument, Item, Service, TimedService, Bundle]
```

Comment:
Part of me wants to DRY up these lists, but I'm not quite sure the best way to do it. We could make a constant `DEFAULT_TYPES = [..]` and then this and mergeable_types could do `@types ||= DEFAULT_TYPES.dup`. I'm also ok on punting on that.